src/app/containers/Timestamp/relativeTimestamp.js

```diff
@@ -0,0 +1,34 @@
+const constructTimestamp = timeDiffMins => {
+  let timeDiff;
+  let unit;
+
+  if (timeDiffMins >= 60) {
+    timeDiff = parseInt(timeDiffMins / 60, 10);
+    unit = timeDiff > 1 ? 'hours' : 'hour';
+  } else {
+    timeDiff = timeDiffMins <= 1 ? 1 : timeDiffMins;
+    unit = timeDiff === 1 ? 'minute' : 'minutes';
+  }
+
+  return `${timeDiff} ${unit} ago`;
+};
```

Comment:
CodeClimate has now reduced complexity from 10 to 8, and is now explicitly complaining about the `constructTimestamp` function. So already this has helped improve things _and_ make things more specific.

If we want all the green ticks, we could work on refactoring this function.

My instinct would be this as a first step:

```js
const suffix = 'ago';
if (timeDiffMins >= 60) {
  const hours = parseInt(timeDiffMins / 60, 10);
  return `${hours} ${hours === 1 ? 'hour' : 'hours'} ${suffix}`;
}
return `${timeDiffMins} ${timeDiffMins === 1 ? 'minute' : 'minutes'} ${suffix}`;
```

...then I'd see the two return statements are extremely similar and maybe move those out to a separate function, should keep CodeClimate happy.

But I'm also happy approving this as-is, think it's readable enough 👍

Reply:
We need line 9
(`timeDiff = timeDiffMins <= 1 ? 1 : timeDiffMins;`)
for this test https://github.com/bbc/simorgh/blob/1a5ffed6d7beedc53bbcfcb8e44c22bb9bc40ffc/src/app/containers/Timestamp/relativeTimestamp.test.js#L71
otherwise we'll get `1 minutes ago` :(